netmanager/src/config/urls/deviceRegistry.js

```diff
@@ -1,59 +1,57 @@
 import { stripTrailingSlash } from '../utils';
 
-const BASE_DEVICE_REGISTRY_URL = stripTrailingSlash(
-  process.env.REACT_APP_BASE_DEVICE_REGISTRY_URL || process.env.REACT_APP_BASE_URL
-);
+const BASE_DEVICE_REGISTRY_URL = stripTrailingSlash(process.env.REACT_APP_BASE_URL);
 
 const BASE_DEVICE_REGISTRY_URL_V2 = stripTrailingSlash(process.env.REACT_APP_BASE_URL_V2);
 
-export const ACTIVITY_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities`;
+export const ACTIVITY_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/activities`;
 
-export const REGISTER_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices`;
+export const REGISTER_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices`;
 
-export const ALL_DEVICES_URI = `${BASE_DEVICE_REGISTRY_URL}/devices`;
+export const ALL_DEVICES_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices`;
 
-export const EDIT_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices`;
+export const EDIT_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices`;
 
-export const SOFT_EDIT_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/soft`;
+export const SOFT_EDIT_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/soft`;
 
-export const DEVICES_IN_LOCATION_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/by/location?loc=`;
+export const DEVICES_IN_LOCATION_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/by/location?loc=`;
 
-export const DEPLOY_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/deploy`;
+export const DEPLOY_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/deploy`; // Change to V2
 
-export const ADD_MAINTENANCE_LOGS_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/maintain`;
+export const ADD_MAINTENANCE_LOGS_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/activities/maintain`; // Change to V2
 
-export const DELETE_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/ts/delete`;
+export const DELETE_DEVICE_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/ts/delete`;
 
-export const ADD_COMPONENT_URI = `${BASE_DEVICE_REGISTRY_URL}/devices/add/components?device=`;
+export const ADD_COMPONENT_URI = `${BASE_DEVICE_REGISTRY_URL_V2}/devices/add/components?device=`;
```

Comment:
components routes need to be removed, we no longer have this functionality in our codebase @Codebmk

Reply:
done